### PR TITLE
Issue #2690131 by Primsi: Add tests asserting the working of the element

### PR DESCRIPTION
**Pull request by primsi**

### CODE REVIEW

File: tests/modules/dropzonejs_test/dropzonejs_test.module [hunk 1]
```diff
@@ -0,0 +1 @@
+<?php
```
Comment (slashrsm) [inline tests/modules/dropzonejs_test/dropzonejs_test.module]: Can we remove this file?

File: tests/modules/dropzonejs_test/src/Form/DropzoneJsTestForm.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?php
+/**
```
Comment (slashrsm) [inline tests/modules/dropzonejs_test/src/Form/DropzoneJsTestForm.php]: @file block is not required any more.

File: tests/src/Kernel/DropzoneJsElementTest.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+   * Tests that the dropzonejs element appears.
+   */
+  public function testDropzoneJsElement() {
+    $this->container->get('router.builder')->rebuild();
+    $form = \Drupal::formBuilder()->getForm('\Drupal\dropzonejs_test\Form\DropzoneJsTestForm');
+    $this->render($form);
+
+    $xpath_base = "/html/body/form/div[contains(@class, 'form-item-dropzonejs')]";
+    // Label.
+    $this->assertFalse($this->xpath("$xpath_base/label[text()='Not DropzoneJs element']"));
```
Comment (slashrsm) [inline tests/src/Kernel/DropzoneJsElementTest.php:47]: What are we testing with this assert?

Comment (primsi) [inline tests/src/Kernel/DropzoneJsElementTest.php:47]: I got some false positives, not sure why so I added this.

**Pull request merged**